tutorials/basic-topics/linear-search.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Linear search'
+description: 'Linear search is a searching technique that works in a linear/sequential fashion. It traverses whole of the data structure and returns the position(s) where the value is found. If not found then it simple returns nothing.'
+hide_table_of_contents: true
+---
+
+<TutorialAuthors names="@siddoinghisjob"/>
+
+## Overview
+
+Lets say we have a linear data structure - array, linked list - and we need to search for a certain element. To search there are mainly two searching techniques, one being linear search. In linear search we traverse the whole array and then while traversing we check for the particular item. If there's a match then we return that position( we can return multiple positions too).
```

Comment:
> To search there are mainly two searching techniques, one being linear search

What is the second one?

> If there's a match then we return that position( we can return multiple positions too).

unwanted extra space and how can you return multiple positions?

Reply:
By array or vector, we can return multiple values. Also we can simply print all the positions.